packages/api/src/car.js

```diff
@@ -270,10 +264,11 @@ export async function sizeOf (response) {
  * - Missing non-root blocks (when root block has links)
  *
  * @typedef {{ size: number, blocks: number }} CarStat
- * @param {Uint8Array} carBytes
+ * @param {Blob} carBlob
  * @returns {Promise<CarStat>}
```

Comment:
nit: Given we only use the dag size, should we now return the unnecessary blocks?

Reply:
I think once the db migration is done, I'd like to track more metrics about CARs we see, like the number of blocks and the sum of the block sizes vs the dag size, things like that. I'm inclined to keep carStat returning more info for now.